### DISCUSSION TITLE
how about to make optional json parameters in the response?

### DISCUSSION THREAD

**Discussion by abr4xas** [spatie/laravel-health#92] [Ideas]
What do you think about that? I think that a good use case is if you want to build a service API and you want to expose a custom endpoint about the health of your app without expose sensitive information. 

```json
{
	"finishedAt": 1651005040,
	"checkResults": [
		{
			"name": "database",
			"shortSummary": "database is up and running",
			"status": "ok",
		},
		{
			"name": "cache",
			"shortSummary": "we are investigating reports of degraded performance for our cache system",
			"status": "error",
		}
	]
}
```

Hope this makes sense to you 😉

**Comment by freekmurze** [comment]
We already have a controller in the package that will output the results as JSON.

**Reply by abr4xas** [reply]
Hey, yeah. I know but that controller has to many sensitive data to expose.  At last for me is very sensitive. 
I have a "work around" that filter the json response to something like the example on my first comment.

Anyway, thanks Freek!! Have a great one :)

**Reply by freekmurze** [reply]
Yeah, I think if you consider the default info a bit too sensitive, you can add your own controller and start from our default implementation and mold it to want you want! 👍
